pkg/agent/util/sriov_utils.go

```diff
@@ -0,0 +1,140 @@
+package util
+
+import (
+        "fmt"
+        "context"
+        "net"
+        "os"
+        "time"
+        "github.com/containernetworking/cni/pkg/skel"
+        "github.com/containernetworking/cni/pkg/types"
+        "k8s.io/kubernetes/pkg/kubelet/util"
+        "k8s.io/kubernetes/pkg/kubelet/apis/podresources"
+        podresourcesv1alpha1 "k8s.io/kubernetes/pkg/kubelet/apis/podresources/v1alpha1"
+        sriov_cni_utils "github.com/k8snetworkplumbingwg/sriov-cni/pkg/utils"
```

Comment:
Our convention is sriovcniutils (even it is hard to read).

Reply:
Done